lib/puppet/functions/is_absolute_path.rb

```diff
@@ -0,0 +1,10 @@
+Puppet::Functions.create_function(:is_absolute_path, Puppet::Functions::InternalFunction) do
+  dispatch :deprecation_gen do
+    scope_param
+    optional_repeated_param 'Any', :args
```

Comment:
@DavidS @HelenCampbell 
I've got failing builds again...

`optional_repeated_param` not available in puppet < 3.8.1 ??
https://github.com/puppetlabs/puppet/commit/c7419c10b409d860334ab0302c5b8da374e039d2